react/SelectBox/SelectBox.jsx

```diff
@@ -60,6 +60,11 @@ const reactSelectControl = CustomControl => ({
   </div>
 )
 
+export const withPrefix = (cx, className) => {
+  const classNameWithPrefix = cx(null, { [` ${className}`]: true })
```

Comment:
Why `null` ?

Reply:
Here is the cx function : https://github.com/JedWatson/react-select/blob/bd3705b/src/utils.js#L42-L55
```
export function classNames(
  prefix?: string | null,
  cssKey?: string | null,
  state?: ClassNamesState,
  className?: string,
) {
  const arr = [cssKey, className];
  if (state && prefix) {
    for (let key in state) {
      if (state.hasOwnProperty(key) && state[key]) {
        arr.push(`${applyPrefixToName(prefix, key)}`);
      }
    }
  }

  return arr.filter(i => i).map(i => String(i).trim()).join(' ');
}
```
The first parameter is bind by react-select [here](https://github.com/JedWatson/react-select/blob/9690b28dd7713b68728e56075d6819e6898ec430/src/Select.js#L692), so in this place `null` is `cssKey`, it's not necessary.